crates/algod_client/src/models/model_box.rs

```diff
@@ -0,0 +1,37 @@
+/*
+ * Algod REST API.
+ *
+ * API endpoint for algod operations.
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * Contact: contact@algorand.com
+ * Generated by: Rust OpenAPI Generator
+ */
+
+use crate::models;
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+
+/// Box name and its content.
+#[serde_as]
+#[derive(Clone, Default, Debug, PartialEq, Serialize, Deserialize)]
+pub struct ModelBox {
```

Comment:
```suggestion
pub struct BoxModel {
```
Alternatively `StoredBox`, `BoxStore`, `BoxDefinition`.

Reply:
@neilcampbell this is now just `Box`, the issue was in the filename itself which is still kept as model_box.rs but the struct itself was renamed back to Box.

Reply:
https://github.com/algorandfoundation/algokit-core/pull/179#discussion_r2176283771